Gemfile

```diff
@@ -63,3 +64,6 @@ group :test do
 end
 gem "pundit", "~> 2.3"
 gem 'pundit-matchers'
+gem 'rack-attack'
```

Comment:
Can you please explain why are we using this gem?

Reply:
Bhaiya, the Rack-Attack gem protects our Rails server from bad clients. It provides some functionality like allow, block, and throttle based on the request. 

I used to protect my login post-route so that bad clients could not crack the password using brute-force attacks. I configure this rack attack in my project so that if a user gives 10 times the wrong password, it will block 5 minutes. At this time, this client cannot send any requests for login. 

Thank you!